docs/en/operation/Journal.md

```diff
@@ -9,29 +9,38 @@ priority: 5
 * Table of Contents
 {:toc}
 
-Alluxio is fault-tolerant: force-killing the system will not lose metadata. To achieve
-this, the master writes edit logs of all metadata changes. On startup,
-a recovering master will read the edit logs to restore itself back to its previous state.
+Alluxio is fault-tolerant: force-killing the system will not lose metadata.
+To achieve this, the master writes edit logs of all metadata changes (e.g., creating files,
```

Comment:
only `the leading master` writes edit logs? since we're talking about fault tolerance, we should make it clear which master type, the leading master vs the backup masters

Reply:
it is more complicated: journal is written even we have a single master. In addition, I want to avoid introducing too many concepts

Reply:
in that case, we could simplify it further by removing any mention of fault tolerance. maybe just jump right into the writing of the journal metadata. for example, start off the doc with:
`Alluxio keeps an edit log of all metadata related changes, such as creating files or renaming directories. This edit log is referred to as the journal. Upon startup, the Alluxio master will replay all the steps recorded in the journal to recover its last saved state. The purpose of this documentation is to help Alluxio administrators understand and manage the Alluxio journal.`

Reply:
SG